Gemfile

```diff
@@ -77,6 +83,7 @@ group :test do
   gem 'simplecov', '~> 0.17.0', require: false
   gem 'webdrivers'
   gem 'webmock'
+  gem 'show_me_the_cookies'
```

Comment:
This is a gem that helps us manage cookies in our system tests. This is specifically used for testing the Remember me function.